careerfairsystems/arkad-app-2018
# Add filter on companies tab and add loading screen

## Change description

- Adding a filter option on the companies tab
- Adding a loading screen

## How to verify

- Navigate to the companies tab and:
  - Confirm that pressing the filter button in the navigation bar takes you to a filter view
  - Confirm that filters are selectable and searchable
  - Confirm that selecting a filter will filter the company list
  - Confirm that searching while a filter is applied works as intended
  - Confirm that pressing clear filters (both in filters screen and on companies screen) will remove the filter
- Confirm that a loading screen is displayed when starting the app
  - Try a faulty URL by changing it in the company.js action and confirm that a error message is shown

## Issues fixed

This fixes #32 and fixes #50.
